main.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+
+let operators = document.querySelectorAll(".operator"),
+  numbers = document.querySelectorAll(".number"),
+  display = document.querySelector(".display"),
+  clearButtons = document.querySelectorAll(".cancel"),
+  result = document.querySelector("#result"),
+  point = document.querySelector("#point"),
+  a = "",
+  b = "",
+  c = "",
+  memoryOperator = "";
+
+for (let i = 0; i < clearButtons.length; i++) {
+  let cancelBtn = clearButtons[i];
```

Comment:
according the code style it's better not to shorten names

Reply:
done)